### ISSUE TITLE
Write-only Realm permissions

### ISSUE THREAD

**Issue by RiverbayChris** [realm/realm-mobile-platform#178]
## Goals

Write-only Realm access

## Expected Results

Apply correct write-only change permission.  Try to write data from account with write-only permissions.

## Actual Results

I get a .clientSessionError and in the log it prints this:
error: sync-server: Connection[24]: Session[1]: Permission denied (message_type='bind', permission='download', path='/837c74068af3600cd0fb710da39aac54/PHARM00001'

Is there a trick to get "write-only" to work?  Or is this a bug?

## Steps to Reproduce

Apply correct write-only change permission to an account and try to write data from account with write-only permissions.

## Version of Realm and Tooling

- Realm Object Server Version: 1.4.1
- Flavor:
  - [X] Developer
  - [ ] Professional
  - [ ] Enterprise
- Server OS & Version: MacOS
- Client SDK Version: Latest
- Client OS & Version: iOS (Swift 3.1)

**Comment by bigfish24** [comment]
No, this is expected right now. We currently don't support write-only permissions. Will make sure to update docs regarding this.
